test/test_cases/send_erc20_from_schain_to_mainnet.py

```diff
@@ -36,8 +36,10 @@ def __init__(self, config):
         super().__init__('Send ERC20 from schain to mainnet', config)
 
     def _prepare(self):
-        amount = 2 * 10 ** 18
-        self.blockchain.recharge_user_wallet(self.config.mainnet_key, self.config.schain_name, amount)
+        sleep( 5 )
+        amountRecharge = 20 * 10 ** 18 # 2 * 10 ** 18
```

Comment:
Here too

Reply:
Done. Rolled back to `2 * 10 ** 18`. It works.